src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.PackageDependencyResolution.targets

```diff
@@ -31,8 +31,9 @@ Copyright (c) .NET Foundation. All rights reserved.
          be stored in a shared directory next to the assets.json file -->
     <ProjectAssetsCacheFile Condition="'$(ProjectAssetsCacheFile)' == ''">$(IntermediateOutputPath)$(MSBuildProjectName).assets.cache</ProjectAssetsCacheFile>
     <ProjectAssetsCacheFile>$([MSBuild]::NormalizePath($(MSBuildProjectDirectory), $(ProjectAssetsCacheFile)))</ProjectAssetsCacheFile>
-    
-    <CopyLocalLockFileAssemblies Condition="'$(CopyLocalLockFileAssemblies)' == ''">true</CopyLocalLockFileAssemblies>
+
+    <CopyLocalLockFileAssemblies Condition="'$(CopyLocalLockFileAssemblies)' == '' and '$(_TargetFrameworkVersionWithoutV)' >= '3.0'">true</CopyLocalLockFileAssemblies>
```

Comment:
It looks like this is comparing the target framework version but ignoring the target framework identifier.  We should be looking at both.

Also, should this be enabled by default for libraries?  My feeling is no.  Also consider whether it should apply to projects targeting .NET Standard.

Reply:
I'll fix.

My feeling about enabling it for libraries was also no.